Lib/posixpath.py

```diff
@@ -207,11 +207,14 @@ def ismount(path):
         parent = join(path, b'..')
     else:
         parent = join(path, '..')
-    parent = realpath(parent)
     try:
         s2 = os.lstat(parent)
-    except (OSError, ValueError):
-        return False
+    except OSError:
```

Comment:
ValueError isn't handled here anymore, is that safe?

Reply:
ValueError is raised in two cases:

* The path is not encodable.
* The path contains the null character.

The original patch was checked above. Adding `/..` does not add new errors.

There are no tests for this. I'll perhaps add them.

Reply:
Actually, there are tests for this. At the end of `test_ismount_non_existent`.